app/controllers/dashboards_controller.rb

```diff
@@ -13,10 +18,28 @@ def refresh_dashboard
 
   private
 
-  def dashboard
+  def active_dashboard
+    if dashboard.user_role == "solicitor"
+      @_active_dashboard ||= dashboard(:not_finished)
+    else
+      @_active_dashboard ||= dashboard
+    end
+  end
+
+  def closed_dashboard
+    @_closed_dashboard ||= dashboard(:finished, :aborted)
+  end
+
+  def set_dashboard_count
+    @active_count ||= active_dashboard.defence_requests.count
+    @closed_count ||= closed_dashboard.defence_requests.count
+  end
+
+  def dashboard(*additional_scopes)
```

Comment:
Where do these additional scopes come from? Any reason why they shouldn't be loaded with the requests scoped by policy?